qiskit_nature/second_q/operators/bosonic_op.py

```diff
@@ -432,15 +432,11 @@ def index_order(self) -> BosonicOp:
         .. note::
 
             You can use this method to achieve the most aggressive simplification of an operator
-            without changing the operation order per index. :meth:`simplify` does *not* reorder the
-            terms and, thus, cannot deduce ``-_0 +_1`` and ``+_1 -_0 +_0 -_0`` to be
-            identical labels. Calling this method will reorder the latter label to
-            ``-_0 +_0 -_0 +_1``, after which :meth:`simplify` will be able to correctly collapse
+            without changing the operation order per index. :meth:`simplify` can only reduce terms
+            with identical label, but does *not* reorder them. Thus, cannot deduce ``-_0 +_1`` and
+            ``+_1 -_0`` to be identical labels. Calling this method will reorder the latter label to
+            ``-_0 +_1``, after which :meth:`simplify` will be able to correctly collapse
             these two labels into one.
```

Comment:
This example now lacks a bit. Maybe we can rewrite it as follows:
```
            You can use this method to achieve the most aggressive simplification of an operator
            without changing the operation order per index. :meth:`simplify` does *not* reorder the
            terms and, thus, cannot deduce ``-_0 +_1`` and ``+_1 -_0`` to be identical labels.
            Calling this method will reorder the latter label to ``-_0 +_1``, correctly collapsing
            these two labels into one.
```

Reply:
Solved